### PR TITLE
ARQ-2086 Test failures not reported with recent versions of TestNG

### LINKED ISSUE #117: testng 6.9.12+
hi
any plan for update testng to 6.9.12 or major?
thanks in advance
regards

### PR DESCRIPTION
**Pull request by bartoszmajsak**
TestNG false positives. Related to #117 

Fixes #117 #128

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
There is a new version released, but not published yet to Maven Central. https://bintray.com/cbeust/maven/testng/6.12

With this I can confirm that all works fine.

We have to wait however until the new version of TestNG lands in Maven Central :(

Temporary workaround to anyone who wants to give it a try is to add private repository of @cbuest to root `pom.xml`

```xml
<repositories>
  <repository>
    <snapshots>
      <enabled>false</enabled>
      </snapshots>
      <id>bintray-cbeust-maven</id>
      <name>bintray</name>
      <url>http://dl.bintray.com/cbeust/maven</url>
    </repository>
</repositories>
```

File: build/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <version.javax.inject>1</version.javax.inject>
     <version.junit>4.12</version.junit>
     <version.mockito_all>1.10.19</version.mockito_all>
-    <version.testng>6.9.10</version.testng>
+    <version.testng>6.11</version.testng>
```
Comment (bartoszmajsak) [inline build/pom.xml]: This needs to be changed to `6.12`. @dipak-pawar can you give me push rights to your repo or to this branch?

Comment (dipak-pawar) [inline build/pom.xml]: @bartoszmajsak done

**Comment by bartoszmajsak** [comment]
Related issue https://github.com/cbeust/testng/issues/1506

**Comment by bartoszmajsak** [comment]
Upstream 5eddd661

**Pull request closed without merging**